packages/paste-website/src/pages/primitives/menu-primitive/index.mdx

```diff
@@ -0,0 +1,257 @@
+---
+title: Menu - Primitives
```

Comment:
```suggestion
title: Menu Primitive
```

Reply:
So intention here for our page titles is that for SEO we follow a "{page} - {section} - {site}" convention. So this is just Menu, in primitives